.ci/filter/filter_podman_test.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+test_config_file="$1"
+
+it_skip_flag="podman.It"
```

Comment:
if we will use --focus to run the tests, we should rename this variable and some others in this file. Maybe something more general like `it_filter_flag`?